### ISSUE TITLE
Rearranging the World Boss material in Ascension costs

### ISSUE THREAD

**Issue by Zerodymegx** [theBowja/genshin-db#135]
The world boss material in ascension costs is the only one that's not necessary in A1 or for Aether/Lumine. 
For e.g. if you query Aether's ascend costs, you get something like this. 

"costs":
{"ascend1":[{"name":"Mora","count":20000},{"name":"Brilliant Diamond Sliver","count":1},{"name":"Windwheel Aster","count":3},{"name":"Damaged Mask","count":3}],
"ascend2":[{"name":"Mora","count":40000},{"name":"Brilliant Diamond Fragment","count":3},{"name":"Windwheel Aster","count":10},{"name":"Damaged Mask","count":15}],
...

Similar query for Amber will return 

"costs":
{"ascend1":[{"name":"Mora","count":20000},{"name":"Agnidus Agate Sliver","count":1},{"name":"Small Lamp Grass","count":3},{"name":"Firm Arrowhead","count":3}],
"ascend2":[{"name":"Mora","count":40000},{"name":"Agnidus Agate Fragment","count":3},{"name":"Everflame Seed","count":2},{"name":"Small Lamp Grass","count":10},{"name":"Firm Arrowhead","count":15}],
....

Note how the Everflame seed(Boss material for Ascension) is placed at 3rd spot. Would it be possible to move it to 5th spot instead? 
This way all the materials placement is consistent. 
Something like : 
"ascend2":[{"name":"Mora","count":40000},{"name":"Agnidus Agate Fragment","count":3},{"name":"Everflame Seed","count":2},{"name":"Small Lamp Grass","count":10},{"name":"Firm Arrowhead","count":15}],
==> 
"ascend2":[{"name":"Mora","count":40000},{"name":"Everflame Seed","count":2},{"name":"Small Lamp Grass","count":10},{"name":"Firm Arrowhead","count":15}],{"name":"Agnidus Agate Fragment","count":3},

**Comment by theBowja** [comment]
I won't rearrange the order. That order is the way it shows up in the game and I would like it keep it in that order.

However I can add an extra property called "type" for each material so you can sort it yourself.

**Comment by Zerodymegx** [comment]
Hey again, adding an extra type actually works even better. 
I am guessing you mean a query like below would work after you add it?
genshindb.characters('amber').costs.ascend1.type1

**Comment by theBowja** [comment]
I mean something like:
```
{
	"ascend2":[
		{
			"name":"Mora",
			"type": "MORA"
			"count":40000
		},
		{
			"name":"Agnidus Agate Fragment",
			"type": "FRAGMENT"
			"count":3
		},
		{
			"name":"Everflame Seed",
			"type": "NORMALBOSSDROPS"
			"count":2
		},
		{
			"name":"Small Lamp Grass",
			"type": "SPECIALTY"
			"count":10
		},
		{
			"name":"Firm Arrowhead",
			"type": "COMMONDROP"
			"count":15
		}
	]
}
```

I haven't decided on the values for the types yet.

**Comment by Zerodymegx** [comment]
Yes this would definitely help. Would you be adding this type for all materials  or just ascension. Anyways either would be pretty helpful, so looking forward to it.

**Comment by theBowja** [comment]
Materials probably needs a rework which I won't be getting around to anytime soon because I'm busy procrastinating.
